src/commands/build.yml

```diff
@@ -72,7 +81,13 @@ steps:
   - run:
       name: <<parameters.step-name>>
       command: |
+        IFS="," read -ra CACHE_FROM_IMAGES \<<< "<< parameters.cache-from >>"
+        for image in "${CACHE_FROM_IMAGES[@]}"; do
+          docker pull $image || echo "Failed to pull the specified image: $image. Ignoring..."
+        done
+
         docker build \
+          --cache-from "<<parameters.cache-from>>" \
```

Comment:
While I appreciate the intent I am not sure this makes much sense for CircleCI. On the container you are currently building on, no other images will be present in the container to use as cache unless you had already previously pulled them in the same job.

Reply:
I posted a comment related to this in CircleCI-Public/aws-ecr-orb#4. I am open to input, shall we continue the discussion there to avoid double posting?